doc/cephfs/mds-states.rst

```diff
@@ -0,0 +1,227 @@
+
+MDS States
+==========
+
+
+The Metadata Server (MDS) goes through several states during normal operation
+in CephFS. For example, some states indicate that the MDS is recovering from a
+failover by a previous instance of the MDS. Here we'll document all of these
+states and include a state diagram to visualize the transitions.
+
+State Descriptions
+------------------
+
+Common states
+~~~~~~~~~~~~~~
+
+
+::
+
+    up:active
+
+This is the normal operating state of the MDS. It indicates that the MDS
+and its rank in the file system is available.
+
+
+::
+
+    up:standby
+
+The MDS is available to takeover for a failed rank (see also :doc:`standby/`).
```

Comment:
Please don't use paths to link to other documents, use the `ref` syntax (https://github.com/ceph/ceph/blob/master/doc/start/documenting-ceph.rst#toc-and-hyperlinks )

Reply:
Fixed! Thanks for the link/review!